src/text.rs

```diff
@@ -0,0 +1,73 @@
+/// A macro for creating a [`Text`] using vec! syntax.
+///
+/// `text!` is similar to the [`vec!`] macro, but it returns a [`Text`] instead of a `Vec`.
+///
+/// # Examples
+///
+/// * Create a [`Text`] containing a vector of [`Line`]s:
+///
+/// ```rust
+/// # use ratatui::prelude::*;
+/// use ratatui_macros::text;
+///
+/// let text = text!["hello", "world"];
+/// let text = text!["hello".red(), "world".red().bold()];
+/// ```
+///
+/// * Create a [`text`] from a given [`Line`] repeated some amount of times:
+///
+/// ```rust
+/// # use ratatui::prelude::*;
+/// use ratatui_macros::text;
+///
+/// let text = text!["hello"; 2];
+/// ```
+///
+/// * Use [`line!`] or [`span!`] macro inside [`text!`] macro.
+///
+/// ```rust
+/// # use ratatui::prelude::*;
+/// use ratatui_macros::{line, text, span};
+///
+/// let text = text![line!["hello", "world"], span!(Modifier::BOLD; "goodbye {}", "world")];
+/// ```
+///
+/// [`Text`]: crate::text::Text
+/// [`Line`]: crate::text::Line
+/// [`Span`]: crate::text::Span
+#[macro_export]
+macro_rules! text {
+    () => {
+        ratatui::text::Text::default()
+    };
+    ($line:expr; $n:expr) => {
+      ratatui::text::Text::from(vec![$line.into(); $n])
```

Comment:
same as https://github.com/ratatui-org/ratatui-macros/pull/29#pullrequestreview-2041957891

`cargo fmt` with some macro specific settings (requires nightly) is likely a good idea in this repo (and CI)

Reply:
I did run it and it doesn't change any of the lines in this PR

<img width="757" alt="image" src="https://github.com/ratatui-org/ratatui-macros/assets/1813121/0d702d9f-e2a4-48d5-86cb-5acd36b44082">

It does make a change that I don't like for the `Fill` macro but I could live with that if we added it to CI.

Reply:
Strange. The indentation is different for this line which seems to be not formatted then.